build.gradle

```diff
@@ -2,13 +2,16 @@
 
 buildscript {
     ext.kotlin_version = '1.3.10'
-    ext.android_components_version = '0.50.0'
     ext.jna_version = '5.2.0'
     ext.android_gradle_plugin_version = '3.5.0'
 
+    // N.B. try to keep these two in sync
+    ext.android_components_version = '23.0.0'
+    ext.glean_version = '21.2.0'
```

Comment:
This will go away once mozilla-mobile/android-components#4985 is merged and the plugin can be used :) The plugin will take care of publishing the correct version in your configuration.